app/src/main/java/io/homeassistant/companion/android/sensors/CarSensorManager.kt

```diff
@@ -135,33 +192,39 @@ class CarSensorManager :
     }
 
     override fun hasSensor(context: Context): Boolean {
-        // TODO: show sensors for automotive (except odometer) once
-        //  we can ask for special automotive permissions in requiredPermissions
-        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O &&
-            !context.packageManager.hasSystemFeature(PackageManager.FEATURE_AUTOMOTIVE) &&
-            BuildConfig.FLAVOR == "full"
+        this.context = context.applicationContext
+
+        return if (isAutomotive) {
+            BuildConfig.FLAVOR == "minimal"
```

Comment:
I vote we dont hide this from the automotive play store build personally, its up to google to allow users to have access to settings at this point. The code should work as is once settings access is granted.

Reply:
If we do that, it should be checked that the sync triggers the notification missing permission correctly (it may need a vehicle extension?)

Reply:
ok in that case it may be out of scope for this PR then :)